applications/vanilla/library/Jobs/ExecuteBatchDeleteDiscussion.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * @copyright 2009-2020 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+use Garden\Schema\Schema;
+use Vanilla\Scheduler\Job\JobExecutionStatus;
+use Vanilla\Scheduler\Job\JobPriority;
+
+/**
+ * Execute the discussion delete queue.
+ */
+class ExecuteBatchDeleteDiscussion implements Vanilla\Scheduler\Job\LocalJobInterface {
```

Comment:
This is just a suggestion, but this class could probably be a little less...wordy.

```suggestion
class DeleteDiscussions implements Vanilla\Scheduler\Job\LocalJobInterface {
```

Also, I know some of our previous jobs don't have namespaces, but it would be good if we started using them (e.g. `Vanilla\Library\Jobs`).